pdfarranger/exporter.py

```diff
@@ -171,6 +171,12 @@ def export(input_files, pages, file_out, mode, mdata):
         new_page = pdf_output.make_indirect(new_page)
 
         pdf_output.pages.append(new_page)
+        # Ensure annotations are copied rather than referenced
+        # https://github.com/pdfarranger/pdfarranger/issues/437
+        if pikepdf.Name.Annots in current_page:
+            pdf_temp = pikepdf.Pdf.new()
+            pdf_temp.pages.append(pdf_input[row.nfile - 1].pages[row.npage - 1])
```

Comment:
Can we use current_page here? If no, why not? Would improve readability imo.

Reply:
@dreua Done